### PR TITLE
[CloudMinds] 10352-Thermal Imager

### PR DESCRIPTION
**Pull request by Kenny-SZ**
This LWM2M Object includes the Thermal Imager information.

### CODE REVIEW

File: 10352.xml [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+			<Description>The Highest Temperature of the thermal imager.</Description>
+		</Item>
+		<Item ID="2">
+			<Name>Lowest Temperature</Name>
+			<Operations>R</Operations>
+			<MultipleInstances>Single</MultipleInstances>
+			<Mandatory>Mandatory</Mandatory>
+			<Type>Float</Type>
+			<RangeEnumeration>-100.0..100.0</RangeEnumeration>
+			<Units>°C</Units>
```
Comment (seanmcilroy29) [inline 10352.xml]: Incorrect Unit submitted - The Units defined in OMA LwM2M Objects are defined by SenML - Sensor Measurement Lists.

File: 10352.xml [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+			<Description>The Lowest Temperature of the thermal imager.</Description>
+		</Item>
+		<Item ID="3">
+			<Name>Average Temperature</Name>
+			<Operations>R</Operations>
+			<MultipleInstances>Single</MultipleInstances>
+			<Mandatory>Mandatory</Mandatory>
+			<Type>Float</Type>
+			<RangeEnumeration>-100.0..100.0</RangeEnumeration>
+			<Units>°C</Units>
```
Comment (seanmcilroy29) [inline 10352.xml]: Incorrect Unit submitted - The Units defined in OMA LwM2M Objects are defined by SenML - Sensor Measurement Lists.

File: 10352.xml [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+			</Description>
+		</Item-->
+		<Item ID="1">
+			<Name>Highest Temperature</Name>
+			<Operations>R</Operations>
+			<MultipleInstances>Single</MultipleInstances>
+			<Mandatory>Mandatory</Mandatory>
+			<Type>Float</Type>
+			<RangeEnumeration>-100.0..100.0</RangeEnumeration>
+			<Units>°C</Units>
```
Comment (seanmcilroy29) [inline 10352.xml]: Incorrect Unit submitted - The Units defined in OMA LwM2M Objects are defined by SenML - Sensor Measurement Lists.

File: 10352.xml (hunk 1 above)
Comment (Kenny-SZ) [reply 10352.xml]: °C, Celsius scale,  the unit of Temperature.

File: 10352.xml (hunk 2 above)
Comment (Kenny-SZ) [reply 10352.xml]: °C, Celsius scale,  the unit of Temperature.

File: 10352.xml (hunk 3 above)
Comment (Kenny-SZ) [reply 10352.xml]: °C, Celsius scale,  the unit of Temperature.

File: 10352.xml (hunk 1 above)
Comment (akeranen) [reply 10352.xml]: That's "Cel" in SenML units

File: 10352.xml (hunk 2 above)
Comment (seanmcilroy29) [reply 10352.xml]: Replace ``°C`` with ``Cel``

File: 10352.xml (hunk 3 above)
Comment (seanmcilroy29) [reply 10352.xml]: Replace ``°C`` with ``Cel``

File: 10352.xml (hunk 1 above)
Comment (Kenny-SZ) [reply 10352.xml]: Thanks for your comments, changed from °C to Cel.

File: 10352.xml (hunk 3 above)
Comment (Kenny-SZ) [reply 10352.xml]: Thanks for your comments, changed from °C to Cel.

File: 10352.xml (hunk 2 above)
Comment (Kenny-SZ) [reply 10352.xml]: Thanks for your comments, changed from °C to Cel.

**Pull request merged**